backend/app/modules/scheduler/MonitoringActor.scala

```diff
@@ -0,0 +1,69 @@
+package modules.scheduler
+
+import akka.actor.{Actor, Props}
+import models.Query
+
+import scala.collection.mutable
+
+object MonitoringActor {
+
+  /**
+    * Create a new instance of monitoring actor
+    */
+  def props = Props[MonitoringActor]
+
+  /**
+    * Message definition
+    */
+  case class StartProcess(query: String)
+  case class UpdateProcess(id: String, status: String)
+
```

Comment:
Remove blank line if this is part of the 'Message definition group' (or add a comment on the ListProcess, but I don't think it's the case)